web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -98,11 +101,11 @@
     on:mouseenter={onMouseEnter}
     on:mouseleave={onMouseLeave}
     tabindex={0}
-    on:click={isMultiSelectState ? onIconClickedHandler : undefined}
+    on:click={isMultiSelectState ? onIconClickedHandler : isStackSlideshow ? thumbnailClickedHandler : undefined}
```

Comment:
This is very hard to read and understand. Can you help refactor this?

Reply:
done

Reply:
Food for thought, but if component is being rendered in the context of a stack slideshow, should the thumbnail ever be a link (and therefore Ctrl+clickable)?